### ISSUE TITLE
Magnet Addon

### ISSUE THREAD

**Issue by rada2x** [Andrew2448/Andrew2448PowersuitAddons#11]
Magnet Addon is not workig even if its enabled

**Comment by Amperthorpe** [comment]
He's probably going to want more info than that.

**Comment by Andrew2448** [comment]
Yes I need more than that. Reopen if you have more to share.

**Comment by ntwitch** [comment]
I was having the same issue, so I tested it in a separate instance to make sure it wasn't a mod interaction.

minecraft.jar:
MC1.5.1 with Forge build 679

Installed mods:
CodeChickenCore 0.8.6.1 (coremod, NEI requirement)
NEI 1.5.2.9 (coremod)
CoFHCore 1.5.1.5 (coremod, Thermal Expansion requirement)
ModularPowerSuits 0.7.0 build 489
MPSA 0.2.3 build 132
Thermal Expansion 2.4.1.0

Steps I took:
In a creative world, I place down a tinker table and gave myself a set of MPS armor.
Added one Elite Battery to my chest plate, max capacity.
Added Magnet module
Placed a redstone energy cell and the new TE charging table
Charged the armor a bit
Made sure a keybind for the magnet module was configured
Tossed out a few items
Toggled keybind several times. Enable/Disable message shows up properly.
Left it enabled for 30+ seconds
Verified it IS using energy from the suit's battery.

No items were drawn to me or picked up at any distance (until I reached normal vanilla item pickup distance of course)
